grpc/sa-wrappers.go

```diff
@@ -189,6 +189,36 @@ func (sac StorageAuthorityClientWrapper) CountCertificatesByNames(ctx context.Co
 	return names, nil
 }
 
+func (sac StorageAuthorityClientWrapper) CountCertificatesByExactNames(ctx context.Context, domains []string, earliest, latest time.Time) (map[string]int, error) {
+	earliestNano := earliest.UnixNano()
+	latestNano := latest.UnixNano()
+
+	response, err := sac.inner.CountCertificatesByExactNames(ctx, &sapb.CountCertificatesByNamesRequest{
+		Names: domains,
+		Range: &sapb.Range{
+			Earliest: &earliestNano,
+			Latest:   &latestNano,
+		},
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	if response == nil || response.CountByNames == nil {
+		return nil, errIncompleteResponse
+	}
+
+	names := make(map[string]int, len(response.CountByNames))
```

Comment:
Great. Also, I realized this was confusing because I said "below", but this came out at the bottom. Looks like GitHub review comments are ordered by time of creation, not by order in the diff. How smart!